README.md

```diff
@@ -149,11 +149,19 @@ dpct is an alias command for c2s.
 ### Run in-tree LIT tests
 SYCLomatic uses [LLVM Integrated Tester](https://llvm.org/docs/CommandGuide/lit.html) infrastructure to do the unit test.
 Note: Certain CUDA header files may need to be accessible to the tool.
-After building the SYCLomatic, you can run the list test by:
+After building the SYCLomatic, you can run the lit test by:
 ```
 ninja check-clang-c2s
 ```
 
+### Run in-tree build LIT tests (Linux Only)
+After building the SYCLomatic, you can run the build lit by
+```
+export BUILD_LIT=TRUE
```

Comment:
"export BUILD_LIT=TRUE" is optional not a must,  and it has extra dependency on sycl compiler. 
So change to:  "
ninja check-clang-c2s  

or 

export BUILD_LIT=TRUE"  # if you want to enable the build test of migrated SYCL files from lit tests.
ninja check-clang-c2s